codice-itest-impl/src/main/java/org/codice/itest/reporter/TestReporterConfiguration.java

```diff
@@ -52,22 +35,21 @@ public ExitCodeReporter exitCodeReporter(){
     }
 
     @Bean("testReporter")
-    @ConditionalOnProperty(prefix="itest.reporter", name="logger")
+    @ConditionalOnProperty(prefix="codice.itest", name="loggerName")
     public Consumer<TestResult> loggingDiagnosticTestReporter() {
-        Logger logger = LoggerFactory.getLogger(loggerName);
-        return new LoggingDiagnosticTestReporter(logger, exitCodeReporter(), toStringFormatter());
+        Logger logger = LoggerFactory.getLogger(iTestConfigurationProperties.loggerName());
+        return new LoggingDiagnosticTestReporter(logger, (tr) -> tr.toString());
```

Comment:
:memo: For the two `LoggingDiagnosticTestReporter` constructor calls in this file, the lamda can be replaced by method reference:

`(tr) -> tr.toString()` can be changed `Object::toString`

Reply:
changed